andela/ah-scorpion
# [Feature #159053986] Add endpoints and functionality for creating articles

**What does this PR do?**
Implements the creating of articles by the users

**Description of Task to be completed?**
- Add a new app called articles
- Create models, serializers and views for articles and migrate models to the database
- Create endpoints that allow the creation of articles

**How should this be manually tested?**
- Clone the repository from [here](https://github.com/andela/ah-scorpion) and check out the branch `ft-159053986-Add-endpoint-for-Creating-Posts`
- Activate your virtual environment and use the `.env-sample` to set the environment variables.
- Use postman to test these endpoints.
  `POST /articles/` for creating an article

{
  
    "title": "How to train your dragon",
    "description": "Ever wonder how?",
    "body": "You have to believe",
    "tagList": ["reactjs", "angularjs", "dragons"],
    "images" : ["imagelink1", imagelink2]
}
- **You need to login first to use the endpoint. Testing without loging in will give an error**




What are the relevant pivotal tracker stories?
[#159053986](https://www.pivotaltracker.com/n/projects/2184741/stories/159053986)
